css/index.css

```diff
@@ -102,16 +139,89 @@ p {
 }
 
 .middle-content .boxes .box {
+    flex-wrap: nowrap;
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+.middle-content2 {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+
+.boxes-about{
+width: 400px;
+}
+
+.boxes-individual{
+    padding: 0 1%;
+    padding-bottom: 20px;  
+    margin: 10px 10px 60px;
+    /* background-color: whitesmoke; */
+    border-bottom: 1px dashed black;
+}
+
+/* .middle-content img{
+    width: 330px;
+} */
+
+button {
+    border-radius: 8px;
+    text-align: center;
+    padding: 10px 20px;
+    border: solid 2px black;
+    /* border-bottom: solid 2px lightgrey; */
+    background: white;
+    margin: 17px 0;
+}
+
+.box1{
```

Comment:
Something like this would be better instead having ten classes
```
/* box colors */
.boxes .box:nth-child(1) {
  background-color: teal;
}
.boxes .box:nth-child(2) {
  background-color: gold;
}
.boxes .box:nth-child(3) {
  background-color: cadetblue;
}
.boxes .box:nth-child(4) {
  background-color: coral;
}
.boxes .box:nth-child(5) {
  background-color: crimson;
}
.boxes .box:nth-child(6) {
  background-color: forestgreen;
}
.boxes .box:nth-child(7) {
  background-color: darkorchid;
}
.boxes .box:nth-child(8) {
  background-color: hotpink;
}
.boxes .box:nth-child(9) {
  background-color: indigo;
}
.boxes .box:nth-child(10) {
  background-color: dodgerblue;
}
```

Reply:
Thank you for this useful advice!